models/public/mixnet-l/mixnet-l.md

```diff
@@ -0,0 +1,75 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. The `mixnet-l` model is one of the
```

Comment:
what does mean "l" suffix in model name?

Reply:
@eaidova according to description in original repository there are tree variations of mixnet model, MixNet-S, MixNet-M and MixNet-L. So, I'd think the postfix `l` (it is lower case letter L, not a number one) is stands for MixNet-L model.